hazelcast-spring/src/test/java/com/hazelcast/spring/TestClientConfigAndSpringDefaults.java

```diff
@@ -57,7 +57,6 @@ public void before() {
     public void testDefaults() {
         ClientConfig defaults = new ClientConfig();
 
-        assertEquals(defaults.isSmartRouting(), clientConfig.isSmartRouting());
```

Comment:
why? should it be changed to assertEquals(defaults.getNetworkingConfig().isSmartRouting(), clientConfig.getNetworkingConfig().isSmartRouting());

Reply:
The line you are describing is already there. Check one line below.